bin/run_injected_rc2_subset.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+
+welcome="RC2_subset nightly processing with source injection"
+underline=$(printf "%-${#welcome}s" | tr ' ' '-')
+echo -e "$welcome\n$underline"
+
+LSST_DISTRIB=$(eups list | grep lsst_distrib | grep setup)
+if [ -n "$LSST_DISTRIB" ]; then
+    echo -e "$LSST_DISTRIB\n"
+else
+    echo "The lsst_distrib package is not set up. Exiting."
+    exit 1
+fi
+LSST_DISTRIB_EUPS=$(awk '{for (i=3; i<=NF; i++) printf "%s ", $i; print ""}' <<< "$LSST_DISTRIB")
+
+set -e  # Exit immediately on failures from this point on.
+
+RC2_SUBSET_EUPS=${RC2_SUBSET_EUPS:-setup}
+echo "[RC2_SUBSET_EUPS] EUPS lsst_distrib: $RC2_SUBSET_EUPS"
+
+RC2_SUBSET_LEVL=${RC2_SUBSET_LEVL:-INFO}
+echo "[RC2_SUBSET_LEVL] Run logging level: $RC2_SUBSET_LEVL"
+
+RC2_SUBSET_PROC=${RC2_SUBSET_PROC:-8}
+echo "[RC2_SUBSET_PROC] Maximum processes: $RC2_SUBSET_PROC"
+
+RC2_SUBSET_ARGS=${RC2_SUBSET_ARGS:-""}
+echo "[RC2_SUBSET_ARGS] Extra config args: $RC2_SUBSET_ARGS"
+
+RC2_SUBSET_REPO=${RC2_SUBSET_REPO:-$RC2_SUBSET_DIR/SMALL_HSC}
+echo "[RC2_SUBSET_REPO] Butler repository: $RC2_SUBSET_REPO"
+
+RC2_SUBSET_COLL=${RC2_SUBSET_COLL:-u/$USER/RC2_subset/nightly}
+echo "[RC2_SUBSET_COLL] Non-injected output collection: $RC2_SUBSET_COLL"
```

Comment:
So, in the spirit of making these new configs line up to make them easier to parse at a glance, how does this read to you?
```
RC2_SUBSET_COLL=${RC2_SUBSET_COLL:-u/$USER/RC2_subset/nightly}
echo "[RC2_SUBSET_COLL] Existing output collection (non-injected): $RC2_SUBSET_COLL"

RC2_SUBSET_INJECTION_CATS=${RC2_SUBSET_INJECTION_CATS:-injection/catalogs/DM-47887/stars}
echo "[RC2_SUBSET_INJECT_CATS] Injection catalog input collection: $RC2_SUBSET_INJECTION_CATS"

RC2_SUBSET_INJECTION_COLL=${RC2_SUBSET_INJECTION_COLL:-u/$USER/RC2_subset/injected_nightly}
echo "[RC2_SUBSET_INJECT_COLL] Injected output CHAINED collection: $RC2_SUBSET_INJECTION_COLL"

RC2_SUBSET_INJECTION_PIPE=${RC2_SUBSET_INJECTION_PIPE:-$DRP_PIPE_DIR/pipelines/HSC/DRP-RC2_subset+injected_deepCoadd.yaml}
echo "[RC2_SUBSET_INJECT_PIPE] Source injection pipeline def YAML: $RC2_SUBSET_INJECTION_PIPE"
```

Reply:
NB: I also shortened the variables to `INJECT` everywhere, making them shorter to type out and use.